src/meta/api/src/schema_api_keys.rs

```diff
@@ -294,3 +296,48 @@ impl KVApiKey for CountTablesKey {
         Ok(CountTablesKey { tenant })
     }
 }
+
+// __fd_table_stage_file/tenant/db_name/table_name/file_name -> TableStageFileInfo
+impl KVApiKey for TableStageFileNameIdent {
+    const PREFIX: &'static str = PREFIX_TABLE_STAGE_FILE;
+
+    fn to_key(&self) -> String {
+        format!(
+            "{}/{}/{}/{}/{}",
+            Self::PREFIX,
+            self.tenant,
+            self.db_name,
```

Comment:
Is it possible to use db_id and table_id for the prefix?
If the user `drop table t1 and create table t1`, `drop table t1` failed to drop the stage meta files in metasrv, I think there may be some issues.